lib/home/widgets/recipes_screen.dart

```diff
@@ -18,26 +18,79 @@ class RecipesScreen extends StatelessWidget {
-                            arguments: index)));
+                            arguments: index),
+                        onLongPress: () => {
+                              showDialog<String>(
+                                  context: context,
+                                  builder: (BuildContext context) =>
+                                      DeleteRecipe(
+                                          recipe: state.recipes[index]))
+                            }));
               }),
           floatingActionButton: Column(
               mainAxisAlignment: MainAxisAlignment.end,
               crossAxisAlignment: CrossAxisAlignment.end,
               children: <Widget>[
-                FloatingActionButton(
-                    child: const Icon(Icons.add),
+                FloatingActionButton.extended(
+                    heroTag: null,
+                    label: const Text("Standard Recipe"),
+                    icon: const Icon(Icons.add),
                     onPressed: () => {
-                          context.read<RecipeBloc>().add(RecipeAdded()),
+                          context.read<RecipeBloc>().add(
+                              const RecipeAdded(recipeType: "Standard Recipe")),
+                          Navigator.pushNamed(context, PageRouter.recipe,
+                              arguments: state.recipes.length),
+                        }),
+                const SizedBox(
+                  height: 10,
+                ),
+                FloatingActionButton.extended(
+                    heroTag: null,
+                    label: const Text("Non-standard Recipe"),
+                    icon: const Icon(Icons.add),
+                    onPressed: () => {
+                          context.read<RecipeBloc>().add(const RecipeAdded(
+                              recipeType: "Non-standard Recipe")),
                           Navigator.pushNamed(context, PageRouter.recipe,
                               arguments: state.recipes.length),
                         })
               ]));
     });
   }
 }
+
+class DeleteRecipe extends StatelessWidget {
+  final Recipe recipe;
+
+  const DeleteRecipe({Key? key, required this.recipe}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    String recipeName = recipe.recipeName.value;
+    return BlocBuilder<RecipeBloc, RecipeState>(builder: (context, state) {
+      return AlertDialog(
+        title: const Text('Delete recipe'),
+        content: Text('Would you like to delete the $recipeName recipe?'),
+        actions: [
+          TextButton(
+            onPressed: () => Navigator.pop(context, 'Cancel'),
+            child: const Text('Cancel'),
+          ),
+          TextButton(
+            onPressed: () => {
+              context.read<RecipeBloc>().add(RecipeDeleted(recipe: recipe)),
+              Navigator.pop(context, 'OK')
+            },
+            child: const Text('OK'),
+          ),
+        ],
+      );
```

Comment:
I *really* like this `AlertDialog` popup as a confirmation prompt before deleting a recipe. I should implement the same in the `Collection` module. 👍